docs/website/src/landing/article-section/article-section.less

```diff
@@ -48,74 +50,80 @@
       color: inherit;
       text-decoration: none;
     }
-
-    @media @xs-sm {
-      flex: 1 1 40%;
-      height: fit-content;
-    }
   }
 
-  &-item {
-    list-style: none;
-    flex: 1 1 0;
-    max-width: 350px;
-
+  &-item{
     padding: 5px;
-    align-items: center;
-    text-align: start;
 
     @media @xs-sm {
-      flex: 1 1 100%;
-    }
-    @media @xs {
-      max-width: 90%;
-      padding: 5px 0;
+      padding: 0;
     }
 
     &-title {
-      height: 3.6rem;
-      color: @landing-blue;
-      overflow: hidden;
-      text-overflow: ellipsis;
-
-      @media @xs-sm {
-        height: 2.4rem;
-      }
-      @media @xs {
-        height: auto;
-      }
-    }
-
-    &-img {
-      min-width: 150px;
-      width: 100%;
-
-      &-container {
-        width: 100%;
-        justify-content: center
-      }
+      font-weight: 500;
+      font-size: 20px;
     }
 
     &-taglist {
       display: flex;
       flex-wrap: wrap;
       list-style-type: none;
+      margin: 10px 0px;  
 
       &-tag {
         flex: 0 0 auto;
         padding: 0 5px 0 0;
-
+        color: @landing-sec-blue;
         font-size: 0.875rem;
+        font-weight: 500;
       }
     }
 
     &-date {
       font-size: 0.875rem;
       opacity: 0.6;
+      height: 1.5rem;
       color: inherit;
+      margin: 20px 0px;
+      text-align: start;
+    }
+
+    .inner-image {
+      min-width: 150px;
+      height: 380px;
+      margin-bottom: 10px;
+    }
+
+    &-img {
```

Comment:
if we make the date separately for all articles after the picture, then we will not need these styles for `.article-item-img` class

Reply:
this class is now used to override esl-image's default height (100%), which messes up layout a bit